libs/langchain/langchain/retrievers/parent_document_retriever.py

```diff
@@ -56,19 +57,47 @@ class ParentDocumentRetriever(MultiVectorRetriever):
             )
     """
 
-    child_splitter: TextSplitter
-    """The text splitter to use to create child documents."""
+    child_splitter: Optional[TextSplitter] = None
+    """The text splitter to use to create child documents.
+    If none, you will have to provide a list of text splitters corresponding
+    to each document when calling `add_documents`."""
 
+    id_key: str = "doc_id"
```

Comment:
I know this is already defined on the parent class and it's better not to redundantly define it here too. But without it, the useful docstring below it is orphaned.